Steps4Impact/Settings/SettingsViewController.swift

```diff
@@ -58,16 +58,17 @@ class SettingsViewController: TableViewController {
     case .logout:
       logout()
     case .deleteAccount:
-      guard let dataSource = dataSource as? SettingsDataSource else { return }
       let alert = AlertViewController()
       alert.title = Strings.Settings.delete
-      alert.isDismissable = false
-      AppController.shared.present(alert: alert, in: self) {
-        dataSource.deleteAccount { [weak self] in
-          self?.logout()
+      alert.body = Strings.Settings.deleteBody
+      alert.add(AlertAction.cancel())
+      alert.add(AlertAction(title: "Delete", style: .destructive, shouldDismiss: false) { [weak self] in
+        AKFCausesService.deleteParticipant(fbid: Facebook.id) { (_) in
           alert.dismiss(animated: true, completion: nil)
```

Comment:
need onMain {} for UI

Reply:
*sigh* you would think I would learn that, I just fixed that for the reload :/